src/axom/sidre/core/Group.hpp

```diff
@@ -109,9 +109,25 @@ class MapCollection;
  * of the indexed group.  None of these methods is marked with "Child".
  *
  * A Group can optionally be created to hold items in a "list format". In
- * this format, any number of child Group or View items can be created with
- * empty strings for their names, and none of the methods that access
- * child items by name or path will return a valid pointer.
+ * this format, any number of child Group or View items can be created to
+ * be held and accessed like a list.  When using this format, the child items
+ * can be accessed only via iterators that iterate in the order that they
+ * were added to the parent Group.
+ *
+ * It is recommended but not required that the items held in the list format
+ * be created without names.  String names may be assigned to these items,
+ * but the names will not be useful for accessing them from their parent
+ * Group, and none of the methods that access child items by name or path will
+ * return a valid pointer.  Unnamed Groups to be held in the list format
+ * should be created using the method createUnnamedGroup, while unnamed
+ * Views should be created by passing an empty string as the path argument
+ * to any of the createView methods.
+ *
+ * For the list format ONLY, if the method creating a Group or View is
```

Comment:
Is this documentation in the Sphinx docs as well?

Reply:
I need to add that.

Reply:
The sphinx docs need a dedicated section on the list format, so I'll create an issue for that.